go.mod

```diff
@@ -8,5 +8,7 @@ require (
 	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/hashicorp/terraform-plugin-sdk v1.16.0
-	gopkg.in/auth0.v4 v4.6.0
+	gopkg.in/auth0.v4 v4.7.0
 )
+
+replace gopkg.in/auth0.v4 => /Users/kagli/code/auth0
```

Comment:
Not sure if you intended to keep this in and remove once the PR has been approved?  Thought I should flag it in case not!